hack/tempo_multitenancy_openshift/README.md

```
@@ -0,0 +1,19 @@
+# Tracing Stack
+Manifests to setup a tracing stack with OpenTelemetry and Tempo + test traces
+
+## Prerequisites
+* installed Tempo Operator
+* installed Red Hat build of OpenTelemetry
+
+## Endpoints
+* Tempo API: https://tempo-tempo1-gateway-obs-mcp-tracing.apps-crc.testing/api/traces/v1/project1/tempo
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)